### DISCUSSION TITLE
Welcome to Vercel CLI Discussions!

### DISCUSSION THREAD

**Discussion by styfle** [vercel/vercel#3874] [General] [answered]
**Update:** As of January 5th 2021, please visit [Community Discussions](https://github.com/orgs/vercel/discussions) to discuss all things Vercel. This Discussions forum is intended only for discussions relating to Vercel CLI & Runtimes.

___

Hello World,

[GitHub Discussions](https://docs.github.com/pt/free-pro-team@latest/discussions) is currently in beta testing and the Vercel community is invited to participate with us! This is a place to ask questions to the community or request features from our team. You can react to comments just like issues and you can even create threads!

- 🐞 If you found a bug, it is best to [submit a support request](https://vercel.com/support/request) to ensure our team resolves your issue.
- ❓ If you have a question, [search the knowledge base](https://vercel.com/knowledge) as well as existing discussions.
- 🆕 Otherwise, start a new discussion.

As a reminder, the [code of conduct](https://github.com/vercel/vercel/blob/master/.github/CODE_OF_CONDUCT.md) still applies for GitHub Discussions. We want to make this a place that everyone feels welcome!

**Comment by paulogdm** [comment, marked as answer]
Welcome everyone to a new experience in our community!


![giphy](https://user-images.githubusercontent.com/7690649/75932584-8f253480-5e56-11ea-9837-0122f7e437e9.gif)

**Reply by Baiyuetribe** [reply]
Hey @becca,
This is a feature that we'd be excited to give a go with our partner community. Would be great if our repo could be added to the private beta as well.
https://github.com/Baiyuetribe/kamiFaka

Thanks!😃

**Reply by Yacie** [reply]
Heavy

**Reply by hansfpc** [reply]
Nice!

**Reply by heliocosta-dev** [reply]
Looking good!

**Reply by harsh778** [reply]
Noicee! Great feature!

**Reply by Davekibh** [reply]
Nice!

**Reply by panteliskassotis** [reply]
cool

**Reply by hoang17** [reply]
nice

**Reply by Yug34** [reply]
Nice feature!

**Reply by emmysteven** [reply]
cool feature

**Reply by danilopinotti** [reply]
Nice

**Reply by Charmve** [reply]
Hy, A nice place to have a great time!

And more excitingly, welcome to my GitHub home @Charmve

Nice to meet you!

**Reply by PranamBhat** [reply]
Wow!

**Reply by marcj1** [reply]
cool

**Reply by JohnCassavetes** [reply]
👍

**Reply by caioagiani** [reply]
Wow!

**Reply by Sehyn** [reply]
Just discovered that, stackoverflow 2.0 but better ?!

**Reply by jnthmota** [reply]
Discovering this now ... Great! 👍🏽

**Reply by jnthmota** [reply]
> Just discovered that, stackoverflow 2.0 but better ?!

can become

**Reply by evnsh** [reply]
wah

**Reply by mamadaliev** [reply]
Nice!

**Reply by 4po** [reply]
?? Waaaw

**Reply by joomoraes** [reply]
Nice

**Reply by rohitvyas524** [reply]
Noice

**Reply by shashinma** [reply]
Nice!

**Reply by rohitvyas524** [reply]
Very cool

**Reply by isnoa** [reply]
NiCe

**Reply by mewzax** [reply]
nice

**Reply by grayman646** [reply]
Cool

**Reply by ryukaizen** [reply]
Awesome!

**Reply by aschenmaker** [reply]
wow

**Reply by alexandre-vl** [reply]
Okayy

**Reply by trungnv61** [reply]
Nice !

**Comment by BoQsc** [comment]
Can any individual activate/enable discussions for their github repo? I can't see how to enable it on my own repository. What's the setup?

**Reply by paulogdm** [reply]
This is a restricted feature and still in Beta. It will be released to all repos in the future.

**Reply by shrinktofit** [reply]
Speed up please! 💯

**Reply by donhuvy** [reply]
test

**Reply by joebudi** [reply]
Issue/Roadmap item here: https://github.com/github/roadmap/issues/104

**Reply by dante-globant** [reply]
Great :+1:

**Reply by rishav-karanjit** [reply]
Great

**Reply by leozz37** [reply]
Great!

**Reply by patrykwozinski** [reply]
Can't wait to get this feature enabled on my repositories.

**Reply by ghost** [reply]
This is beta stage, will update soon!.

**Reply by 4po** [reply]
hey

**Reply by acarfx** [reply]
Greatt!

**Comment by Gizmotronn** [comment]
Is there a chance to add labels to the discussions? Also possibly an rss feed for the discussions?

**Reply by styfle** [reply]
I believe the GitHub team is adding labels to discussions in the future.

I'm not sure about RSS. I'll forward your feedback to the GitHub team, thanks!

**Reply by Gizmotronn** [reply]
Thanks! The rss feed would be useful for integration with other tools; my team is creating a dashboard to keep track of all the discussions on various boards/forums, and we use rss for this. :)

**Reply by hansfpc** [reply]
It will be great

**Reply by suvrashaw** [reply]
It will be great

**Reply by mxrch** [reply]
Yes, would be cool.

**Reply by 4po** [reply]
Tien tien leaf

**Reply by 4po** [reply]
discord.gg/apo

**Reply by bunnydevx** [reply]
i believe they are working on this!

**Reply by R0Y15** [reply]
I read it somewhere, can't recall at the moment where but read that GitHub developers are soon going up for a very fresh new UI for the GitHub website, though I'm not sure how accurate this news is, but incase this happens someday we'll definitely expect a bunch of new features from the DEV team.

**Comment by martinamorris** [comment]
It would be great to have Discussions enabled for organization-wide purposes, not just individual repos.  

Our use case is a set of packages we develop for network analysis and simulation, [statnet](https://github.com/statnet/).  We have an old-fashioned mailman mailing list for our user help platform now.  Most of these questions are not repo-specific, but relate to the use of the software in empirical application settings.  We would love something like the Discussions platform for this purpose.

**Reply by lubber-de** [reply]
Is it possible to convert existing issues (which were previously used as kind of discussions/questions) into discussions?

**Reply by emmysteven** [reply]
AFAIK it is not being implemented at the moment. 
It is a cool feature **GitHub** might add later, you never can tell.

**Reply by 4po** [reply]
I think the same it's a cool feature for Github

**Comment by andrey-helldar** [comment]
Hi!
I found how to enable discussions for the organization, but I couldn't find how to enable them for the repository.
I saw that this is possible (https://github.com/laravel/framework/discussions).
Tell me how to enable discussions for an organization repository (https://github.com/Laravel-Lang/lang)?
Thanks.

**Reply by becca** [reply]
👋 hey there, @becca from GitHub here. We're still in a limited public beta at the moment but hope to be launching soon as a public beta so that you can opt-in your repos to the Discussions feature! 

I flagged the `Laravel-Lang/lang` repo into the beta so you should be able to toggle Discussions on now there under the `/settings`. You should also have received an invite to our private feedback repo 😄

**Reply by andrey-helldar** [reply]
Thank you! 😊

**Reply by sheldonhull** [reply]
Is there any eta for organizations having access? I'd like to enable on a couple repos but it's not showing up in anything except my personal non-org repos. thanks!

**Reply by ghost** [reply]
Thank you for launching this. We are excited.

**Comment by kapitzas** [comment]
Hi there,

We are soon launching a software package and have collated learning resources on GitHub in this repository: `RangeShifter/RangeshiftR-tutorials`. Would it be possible to enable the Discussions beta on that repo?

Cheers!

**Reply by becca** [reply]
just flagged your repo in.

**Reply by rutkuli** [reply]
lol

**Comment by st-pasha** [comment]
Hi @becca, we were looking for a convenient way to discuss our project (`h2oai/datatable`), and came across this new GitHub feature. Would it be possible to let us join the beta, please?

**Reply by becca** [reply]
just flagged your repo in 🚀

**Reply by st-pasha** [reply]
Thanks, Becca!

**Comment by lubber-de** [comment]
We are also very interested to convert all of our user question issues into github discussions . Maybe @becca you can flag https://github.com/fomantic/Fomantic-UI as well 😉

**Reply by becca** [reply]
done! toggle Discussions on whenever you're ready 🙌

**Reply by lubber-de** [reply]
Thanks a lot!

**Comment by sysarcher** [comment]
@becca any ETA on when this feature is available for private repos and orgs?

**Reply by becca** [reply]
Not at the moment, sorry!

**Comment by YairHalberstadt** [comment]
@becca Any idea what permissions are needed to convert an issue into a discussion? Is triage enough, or is write necessary?

Thanks

**Reply by becca** [reply]
It should be triage+

**Comment by eser** [comment]
@becca Hello,

We're a GitHub-originated non-profit organization named "acik kaynak" (means "open source" in Turkish). Our mission is simply helping people to meet with open source community and guiding them while they are looking for some projects to contribute.

We believe that GitHub Discussions will boost our community and look forward to using discussions. We'd appreciate if you could enable discussions for acikkaynak/acikkaynak.

Regards,

https://github.com/acikkaynak
https://acik-kaynak.org

**Reply by becca** [reply]
👋 hello! you should now see an option to toggle `Discussions` on under the `/acikkaynak/acikkaynak` `/settings` page 🚀

**Reply by eggebraa** [reply]
@becca It looks like this may be the place to request access to testing the Discussions beta feature?  If so, our team is very interested in trying it out.  We have a number of repositories, but the one we'd like to try it on first would be: https://github.com/Alvearie/quality-measure-and-cohort-service

Thanks you!

**Comment by gdcmarinho** [comment]
Pretty nice! I really liked this new feature, it will help a lot in the discussion of open source projects.

**Reply by shashinma** [reply]
The rejoinder to the Stack Overflow.

**Comment by taligentx** [comment]
Hi @becca - if you're looking for additional feedback, it'd be great to test Discussions for https://github.com/taligentx/dscKeybusInterface - we often have long running discussions under Issues that would be much better handled this way, and it's small enough community that beta testing poses no problems and would be glad to contribute feedback.  Thanks!

**Reply by becca** [reply]
👋 would love to see how you start using Discussions for your community. you should now have the option to toggle this feature on under your repo's settings 🚀

**Comment by wabson** [comment]
Hi there, I would really like to have discussions enabled for my public repo [wabson/chafon-rfid](https://github.com/wabson/chafon-rfid) to help resolve issues that some users have had with the underlying hardware that the code integrates with. Right now these tend to land in the Issues section or come via email, neither of which is ideal. Thanks.

**Reply by keshavsingh3197** [reply]
yes now discussions is is enabled for repos.

**Reply by wabson** [reply]
Hi @keshavsingh4522 @becca I don't see the option in the settings for repo [wabson/chafon-rfid](https://github.com/wabson/chafon-rfid) unfortunately, can you confirm is it definitely enabled?

**Comment by 546f6d42** [comment]
Hello !

**Comment by YairHalberstadt** [comment]
Hi

It's possible to convert an issue to a discussion.
I was just wondering if there's any way to convert a discussion to an issue?

Thanks

**Reply by Jkytol** [reply]
I'd like to know that too.

Thanks

**Reply by becca** [reply]
Not yet, no. You can reference a discussion in an issue.
<img width="192" alt="Screen Shot 2020-11-02 at 12 53 32 PM" src="https://user-images.githubusercontent.com/23545426/97907143-71241280-1d0a-11eb-9d91-f16376ce61e0.png">

**Comment by darjoo** [comment]
Hey @becca,  
This is a feature that we'd be excited to give a go with our partner community. Would be great if our repo could be added to the private beta as well.
https://github.com/microsoft/ALAppExtensions

Thanks!

**Reply by becca** [reply]
I flagged your repo in so you can now toggle Discussions on under the repo's `/settings` tab 🚀

**Comment by wabson** [comment]
Hi @becca, is it possible to flag my repo [wabson/chafon-rfid](https://github.com/wabson/chafon-rfid) please. Sorry if you already saw my post above, but I neglected to @mention you in the original request above.

**Comment by kvedurmu** [comment]
Hey @becca, we'd love to try out Github Discussions in our open source project, [Paketo Buildpacks](https://github.com/paketo-buildpacks). Would it be possible to enable this for the [paketo-buildpacks/feedback](https://github.com/paketo-buildpacks/feedback) repo?

Thanks!

**Comment by ilhan007** [comment]
Hey @becca, could you enable Github Discussions for the [UI5 Web Components](https://github.com/SAP/ui5-webcomponents) project?

Thanks!

**Reply by ilhan007** [reply]
Hello @becca I think you missed the UI5 Web Components project?

**Comment by semoal** [comment]
Hey 🙌 @becca we would love discussions on Lingui Library, an amazing internationalization library 
https://github.com/lingui/js-lingui

**Reply by becca** [reply]
my apologies for the delay @semoal. I flagged this repo into the limited public beta so you can toggle Discussions on as soon as you're ready and get started 💥

**Comment by krdn** [comment]
This is a great feature. I also want to beta test. Please add.

https://github.com/HyenaForensic

Thanks!

**Reply by becca** [reply]
Discussions is at the repository level so you'll have to let me know exactly which repository you're looking to target

**Comment by wabson** [comment]
It looks like Github stopped adding projects to the beta, but the [roadmap item](https://github.com/github/roadmap/issues/104) suggests this is still on track to be delivered by December so I'm hopeful we might see this becoming GA soon

**Comment by droshev** [comment]
@becca can you enable the feature for [Fundamental Library Styles](https://github.com/SAP/fundamental-styles) and [Fundamental Library for Angular](https://github.com/SAP/fundamental-ngx)? Thank you

**Reply by becca** [reply]
yes, i can! i flagged those two repositories in so you can toggle Discussions on and get started asap 🎸

**Comment by xrkffgg** [comment]
Look forward to this feature

**Comment by Debug-pro-dev** [comment]
After creating a repository in an organization, I saw an option about discussions. Can they be used already?

**Reply by Xerkus** [reply]
@Debug-pro-dev that was team discussions option probably, a different feature only available to org members

**Reply by Debug-pro-dev** [reply]
@Xerkus 
![image](https://user-images.githubusercontent.com/63473508/99722156-4b945880-2ac1-11eb-8a39-1bc9f1dc23e3.png)

**Comment by edoardottt** [comment]
WOOOOW. So cool :P

**Comment by alvarobartt** [comment]
Hi @becca! I was wondering if you could flag the following repository in, as discussions would ease a lot of conversation that is currently being moved to Jitter or Stackoverflow, as well as helping with reducing the redundant issues already answered, etc. I'd love to try it if possible! 👍🏻 

Repository -> `alvarobartt/investpy`

Keep up the great work! 💪🏻

**Reply by becca** [reply]
hey there @alvarobartt! i just flagged your repository into the beta so you can go ahead and toggle Discussions on to start using it 🚀

**Reply by alvarobartt** [reply]
Thanks a lot @becca! Keep it up with the great work 💪🏻

**Comment by lion152** [comment]
Hey @becca . Is it possible to test out GitHub discussions in our [repository](https://github.com/scandipwa/base-theme)? [We](https://scandipwa.com/) are free and open Source PWA theme for Magento 2, and this could ease communication within our community. We have a lot of communication in Slack which is not helpful as you can't easily find answers posted in Slack.

Will be waiting to try out Discussions with anticipation.

**Reply by becca** [reply]
apologies for making you wait so long! I flagged your repository in so you can toggle Discussions right now ❣️

**Comment by maziyarpanahi** [comment]
Hi @becca, I would appreciate it if you can add [Spark NLP](https://github.com/JohnSnowLabs/spark-nlp) to your beta users. We have thousands of users on Slack channels but the history over there overrides itself. This feature would be great for our users to use the previous discussions/guides/questions. 
Many thanks 👍🏼

**Reply by becca** [reply]
👋 sounds like a great candidate! i just flagged that repo in so you can toggle Discussions on and get started whenever you're ready

**Comment by sheldonhull** [comment]
I've been following for updates, but still not certain if there is a projected date. I'd like to try discussions out with some FAQ work related to a repo in my org. Is there a form or someway to submit a private org repo into getting access for this or be patient and keep waiting?

**Comment by kedi** [comment]
Hey @becca!
I would be pleased if [this](https://github.com/kedi/discordwrapper) could be added to beta users.

Thanks in advance!

**Comment by JarvisCraft** [comment]
Hi there @becca, could you please tell if there are any minimal requirements for those who would like to join beta?
And if it is possible to try it for [this](https://github.com/JarvisCraft/padla) and [this](https://github.com/divine-craft/custom-stuff) repositories?

**Reply by JarvisCraft** [reply]
Just got this feature enabled, thanks a lot :)

**Comment by Fenrirthviti** [comment]
Hey @becca! We at the [OBS Project](https://github.com/obsproject/obs-studio) would love to start testing this for more open development discussion!

**Comment by Davekibh** [comment]
Can we delete a Discussion!

**Reply by JarvisCraft** [reply]
Yep, discussions contain a <kbd>Delete discussion</kbd> button
![image](https://user-images.githubusercontent.com/7693005/102114687-8fbe1300-3e4b-11eb-84b3-60ef31282466.png)

**Reply by panteliskassotis** [reply]
Yes.

**Reply by caioagiani** [reply]
Yes.

**Comment by philsward** [comment]
So an Issue can be converted to a discussion, but I can't find anything that says if a discussion can be converted to an issue. Anybody know?

**Reply by emmysteven** [reply]
Nope a discussion is just for discussion

**Reply by philsward** [reply]
Interesting...

Thanks @emmysteven 

Odd, considering a lot of discussions turn into issues. They have to start somewhere 😉

**Reply by Xerkus** [reply]
@philsward they turn into issues only if involved parties can't behave :stuck_out_tongue:

**Comment by jonlepage** [comment]
hey thanks , too much spaces ,margins paddings... ! , can you guys get inspired from `Slack` and `Discord` 
hard to read big tread workflow . 
But you are on the good track !

Keep dev this, it a good evolutions for Github, but you need do better than Discord and Slack, **you are Microsoft** (think big). 😁
_Tips: we like concatenates and simples things, visuals tis good but dev's like `concatenates` thing ._
make it all easier to read on long streams, pull off padding and margin  , we no need ... 😉 i think .

My mouse wheel is on fire just for read some treads here ! :)
May add icons and interaction on mouse hover ? what do you all think ?

save big 28 pixel ! 
![image](https://user-images.githubusercontent.com/24865815/105280749-81fd5b00-5b78-11eb-89d5-3cc41ba5b426.png)

**Comment by MianJawadAhmad** [comment]
is it still in beta or we can use it in our private repos?

**Reply by jimmydbox** [reply]
Would love to know this. And if it's still in beta, when will we be able to use it in private repos?

**Comment by ummahusla** [comment]
Nice

**Reply by mrismanaziz** [reply]
yes its very cool

**Comment by caioagiani** [comment]
How is it displayed on the profile? need to create discussion topics in the repositories or just reply?

**Comment by haroldao** [comment]
Great 📈✅

**Reply by felipelube** [reply]
This is helpful!

**Reply by 4po** [reply]
<3

**Comment by rat** [comment]
I love this

**Comment by pablobion** [comment]
so nice

**Comment by fu-sen** [comment]
Hahahahahaha! 😆
I know the past ZEIT Now forums. It has changed a few times and now ▲Vercel Discussions are in this place.

I recently made a static site `vercel --prod` and was surprised that the web response was very fast! When did you realize it!? 💻😲
There were people who got it via Cloudflare in the past, but now it is enough to operate only ▲Vercel.
I firmly feel the evolution of ▲ and will continue to see it in the future. 😄

**Reply by fu-sen** [reply]
Link my recently published project:
https://github.com/fu-sen/Vercel-MkDocs

There is no MkDocs in the [New Project Template](https://vercel.com/new/templates), but I'm aware that the Vercel members probably didn't. 
You need to install the pip package to use MkDocs properly. Now you can do both `vercel dev` and` vercel --prod`
I would be happy if MkDocs users started using Vercel through this project.
Many of them relied on Docker. Vercel is easier to use!

**Comment by PranamBhat** [comment]
Wow! That's great news!!

**Comment by umutfahriozkan** [comment]
Wow is omg feature

**Comment by mrismanaziz** [comment]
wow its cool

**Reply by caioagiani** [reply]
Yes

**Comment by caioagiani** [comment]
Very cool

**Reply by lucioerlan** [reply]
yes so cool  💃

**Comment by Saleh7** [comment]
**Pretty nice!** I really liked this new feature, it will help a lot in the discussion of **open source projects**.

**Reply by joomoraes** [reply]
Nice <3

**Comment by demkom58** [comment]
Awesome))

**Reply by joomoraes** [reply]
Nice <3

**Comment by mamadaliev** [comment]
Great!

**Reply by joomoraes** [reply]
Nice <3

**Comment by evnsh** [comment]
cool

**Reply by joomoraes** [reply]
Nice <3

**Reply by RobertGoodman08** [reply]
ok

**Comment by johan-naizu** [comment]
Cool

**Reply by 4po** [reply]
cool what?

**Comment by 4po** [comment]
Very very good functionality added to **Github thanks** to the Git team!


<p align="center"><img src="https://i.imgur.com/LAajnm0.gif" width="2000">

**Reply by joomoraes** [reply]
Hey <3

**Comment by himanshuchandola-zz** [comment]
Good addition to Github. Discussions are important in open source community

**Reply by joomoraes** [reply]
hey <3

**Reply by lesongvi** [reply]
awesome

**Comment by PranamBhat** [comment]
Wow!!!

**Comment by devpolo** [comment]
Yeah! I really liked this new feature!

**Comment by RobertGoodman08** [comment]
Awesome))
